src/pages/ProductDetailPage.tsx

```diff
@@ -0,0 +1,199 @@
+﻿import { Suspense, useState } from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
+import Layout from '@/Layout'
+import ErrorBoundary from '@/components/ErrorBoundary'
+import type { ProductAnnouncementItem } from '@/type'
+import {
+  useProductQuery,
+  useProductDetailQuery,
+  useHighlightReviewQuery,
+  useWishCountQuery,
+  useWishMutation,
+} from '@/api/product'
+import {
+  Container,
+  ProductImage,
+  Name,
+  Brand,
+  Price,
+  TabBar,
+  TabButton,
+  ContentArea,
+  ReviewCard,
+  ReviewAuthor,
+  ReviewContent,
+  EmptyText,
+  InfoList,
+  InfoItem,
+  StickyBar,
+  LikeSection,
+  LikeButton,
+  LikeCount,
+  OrderButton,
+} from '@/styles/ProductDetailPage.styles'
```

Comment:
import문이 상당히 길어지네요! 요런 경우에는

```suggestion
import * as Styles from '@/styles/ProductDetailPage.styles'
```
이렇게 한번에 import 후 Styles.Container 처럼 사용할 수도 있어용!

Reply:
오 잊고 있었습니다! 감사합니다

Reply:
ProductDetailPage.tsx:180 ReferenceError: Container is not defined
    at DetailContent (ProductDetailPage.tsx:105:6)


The above error occurred in the <DetailContent> component.

React will try to recreate this component tree from scratch using the error boundary you provided, ErrorBoundary.
<DetailContent>		
ProductDetailPage	@	ProductDetailPage.tsx:180
<ProductDetailPage>		
(anonymous)	@	router.tsx:22

이런 에러가 떠서 직접 들고 올 때 뜨지 않아 일단 * 대신 기존 방식으로 PR 올렸습니다 ㅠ